apidoc/Titanium/UI/Window.yml

```
@@ -1152,6 +1152,17 @@ properties:
     permission: read-only
     platforms: [iphone, ipad]
 
+  - name: noLimits
+    summary: Adds the FLAG_LAYOUT_NO_LIMITS to the window.
+    description: |
+        In combination with `extendSafeArea` it will extend the window below the statusbar but also
+        set the statusbar to completly transparent.
+    type: Boolean
+    default: {android: false}
```

We don't have a set doc standard for what `default` values should be, but generally we try to use a literal default value, or some explanatory text if it can't be a literal, or differs by platform.
```suggestion
    default: `false`
```

---

I've used it from https://github.com/appcelerator/titanium_mobile/blob/5989a607c4dc8adee1d2d888e5427de4d52b055d/apidoc/Titanium/UI/Window.yml#L926

the other `default: false` values are without ` ` `, so shoud I use it here with or without?

---

Ah, we need to fix this as it shows incorrectly https://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.Window-property-extendSafeArea

---

https://github.com/appcelerator/titanium_mobile/pull/11924 like this?